packages/tests/esm/graphql-codegen-joist.cjs

```diff
@@ -0,0 +1,5 @@
+const mappers = { Author: "src/entities#Author", Book: "src/entities#Book" };
```

Comment:
I'm not sure if we need to use extensions here or not. It depends on how ts-poet in the `graphql-typescript-simple-resolvers` handles it. I will need to test it to determine.